### DISCUSSION TITLE
How to fix issues where calls or jmp are skipped at some point during debugging

### DISCUSSION THREAD

**Discussion by ghost** [x64dbg/x64dbg#3319] [Q&A]
I am new to debugging,

According to an example, I am debugging a program that displays the Hello World message box. Even if I continue to step into or step over from some point, a problem occurs where the program moves to the entry point, ignores the call, and runs and ends. This problem occurs when I turn it on and off. It's resolved, but it keeps waking up again after 5 minutes. Please tell me how to solve it.

https://github.com/x64dbg/x64dbg/assets/145145458/2399dfeb-66cb-4b26-bad4-34d784156fa0

**Comment by mrexodia** [comment]
Are you on arm64 perhaps?

**Reply by ghost** [reply]
i am using amd

**Reply by mrexodia** [reply]
I have no idea, this obviously doesn’t happen on my AMD machine. What is your full configuration and are you running in a VM/anti-cheat loaded etc?

**Reply by ghost** [reply]
CPU virtualization is off, debugger settings are default, and Vanguard is the only anti-cheat.

[program](https://github.com/reversecore/book/tree/master/%EC%8B%A4%EC%8A%B5%EC%98%88%EC%A0%9C/01_%EA%B8%B0%EC%B4%88_%EB%A6%AC%EB%B2%84%EC%8B%B1/01_Reversing_Story/bin)

**Reply by ghost** [reply]
Log after error occurred

DLL Unloaded: [05390000](x64dbg://localhost/address32#05390000) wintypes.dll
Thread 8576 created, Entry: ntdll.[771F5930](x64dbg://localhost/address32#771F5930), Parameter: [005A2C98](x64dbg://localhost/address32#005A2C98)
Thread 12416 created, Entry: ntdll.[771F5930](x64dbg://localhost/address32#771F5930), Parameter: [005A2C98](x64dbg://localhost/address32#005A2C98)

**Reply by mrexodia** [reply]
You might be able to fix this by switching the debug engine to GleeBug.
